css/modals/virtual-background/_virtual-background.scss

```diff
@@ -11,12 +11,12 @@
         opacity: .5;
         border: 2px solid #99bbf3;
         @media (min-width: 432px) and (min-width: 432px) and (max-width: 632px) {
-            height: 56px;
-            width: 56px;
+            height: 60px;
+            width: 60px;
         }
         @media (max-width: 432px){
```

Comment:
Biggest nit of my life: empty space between opening brackets 😅 (for consistency)